src/views/pages/globalping.html

```diff
@@ -1874,8 +1855,14 @@
 
 					// should be after the mainOptions.type was set
 					this.set('realTimeTestResResponse', testRes);
-				}).catch(() => {
+				}).catch((e) => {
 					clearInterval(testReqInterval);
+
+					if (e && typeof e === 'string' && e.toLowerCase() === 'not found') {
```

Comment:
This will likely break soon as we change the message format. The check should be for 404 status, not the text.